### PR TITLE
Adjust seasonal colors of certain flower furnitures to improve visibility

### PR DESCRIPTION
**Pull request by miso-nikomi-nasu**
Since I can't speak English, this sentence has been translated through google.
I'm sorry if there is something strange in the text.
I am sorry if there is something strange because this is my first pull request.

### How to use
It is terrain data.
Correction of inadequate adjustments and errors in the console display.

#### Summary

```SUMMARY: Bugfixes "Terrain adjustment'"```

#### Purpose of change
Most of the correction and adjustment of the terrain setting mistakes that are often seen mainly in forest areas.
In addition, by changing the color of the furniture, such as flowers, it improves visual intelligibility.

#### Describe the solution
Current terrain(Animated gif)
https://i.imgur.com/aS8T4ic.gif

"t_tree_mulberry",
"t_tree_elderberry",
"t_tree_juniper",
When it can actually be harvested,
The color settings that indicate harvestable are not engaged.

"t_tree_pine",
"t_tree_pine_harvested",
"t_tree_blackjack",
"t_tree_blackjack_harvested",
"t_shrub_blackberry",
"t_shrub_blackberry_harvested",
"t_shrub_peanut",
"t_shrub_peanut_harvested",
Color change is not set before and after harvesting, so it is indistinguishable.

"t_fern",
"t_fern_harvested",
However, it is set to reach the fruitful season in spring.
It is set to become a color that can be harvested in summer.
Color change is not set before and after harvesting, so it is indistinguishable.
Since the color and symbol are the same as "t_shrub", they cannot be distinguished.
A tactic that guides zombies over "t_shrub" (movement cost 400) and fights,
"t_fern" (movement cost 150) is a visual hindrance to the player.

"t_tree_birch",
Color setting is indistinguishable from general fruit trees.
In the past, there were few types of trees, and "light_green" was fine.
Now that various trees have increased, it has become difficult to understand visually.

"t_tree",
"t_tree_chestnut",
"t_tree_walnut",
"t_tree_beech",
"t_tree_hazelnut",
("t_tree_cottonwood",) I can't harvest anything
("t_tree_elm",) I can't harvest anything
Will be harvestable in the fall.
The color is "brown_green" and is indistinguishable.
It is a personal opinion, but is it better to color code?

"t_tree_cottonwood",
"t_tree_elm",
An ordinary tree with no harvest set,
It is set to a color that can be harvested in the fall.

"t_tree_hickory_dead",
If the color is "green" it will still appear alive,
Should be fixed.

In ornamental plants
"t_shrub_hydrangea",
"t_shrub_lilac",
The color setting of is indistinguishable from other plants that produce fruits, or is a color that seems to be mistaken.

"t_grass_tall",
This is a personal opinion.
Thanks to the color settings, it is difficult to distinguish from plants that produce fruits.

"f_black_eyed_susan",
This is a personal opinion.
The color setting is the same as "f_dandelion", so it's hard to see.
I think that it is "f_dandelion" and go to picking, and I am disappointed when it is different.
It is not necessary to change it forcibly. Should I change it?

"f_flower_tulip",
"f_lily",
"f_dahlia",
This is a personal opinion.
All are "magenta" and are indistinguishable.
It is not necessary to change it forcibly. Should I change it?

#### Describe alternatives you've considered
Proposed terrain(Animated gif)
https://i.imgur.com/hQfXrif.gif

Summarize only the parts that should be reported

"t_fern",
"t_fern_harvested",
Considering the same movement cost as "t_grass_long" (movement cost 150),
Change the symbol to ",".

The color settings have been greatly changed to prioritize sensory clarity.

"t_fern_harvested" changed "looks_like" to "t_grass_long".

"t_grass_tall",
Change the symbol to "'"
By making the color "green", it looks like "t_grass_long" has grown as it is.

I think it ’s easier to see than before.

#### Testing
Adapt the modified json,
A startup test was performed using "0.D-8891-gb0ce923".

### CODE REVIEW

File: data/json/furniture_and_terrain/terrain-flora.json [hunk 1]
```diff
@@ -2055,9 +2055,9 @@
     "id": "t_fern",
     "name": "fern",
     "description": "A short plant with thousands of leaves which grow in a fractal pattern from the stem.",
-    "symbol": "#",
+    "symbol": ",",
     "looks_like": "t_underbrush",
-    "color": [ "green", "brown_green", "brown", "brown" ],
+    "color": [ "brown_green", "green",  "green", "green" ],
```
Comment (fungusakafungus) [inline data/json/furniture_and_terrain/terrain-flora.json]: The pull request checker dislikes the additional space here.

**Comment by fungusakafungus** [comment]
Great work! Unfortunately, I do not play console version, but this pull request looks very well researched and documented.

**Comment by KorGgenT** [comment]
for the .json files you changed, you need to paste the contents onto [this webpage](http://dev.narc.ro/cataclysm/format.html) and then copy it back into the file. it will format the data so it will pass the JSON validation test.

File: data/json/furniture_and_terrain/terrain-flora.json (hunk 1 above)
Comment (miso-nikomi-nasu) [reply data/json/furniture_and_terrain/terrain-flora.json]: I was at a loss as to what was wrong.
Wasn't there extra space?
Thank you very much.

**Comment by miso-nikomi-nasu** [comment]
@KorGgenT 
Thank you for the helpful information.

**Pull request merged**